packages/cmr-client/src/CMR.ts

```diff
@@ -37,45 +29,91 @@ const userIpAddress = (): Promise<string> =>
  * @private
  */
```

Comment:
CMR.updateToken needs to be refactored.  Suggest to add a new class file (e.g. EarthdataLog.ts) to have method getToken, createToken, revokeToken.

Reply:
suggest to move type definitiond outside of the method, and have a meaningful name.
see example, https://github.com/nasa/cumulus/blob/master/packages/oauth-client/src/OAuthClient.ts#L11

Reply:
If you write integration test, you want to delete token you created during test.